changelogs/fragments/50861-docker_swarm_service-documentation-fixes.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - "docker_swarm_service - Document minimal api_version for ``configs`` and ``secrets``."
```

```suggestion
  - "docker_swarm_service - Document minimal API version for ``configs`` and ``secrets``."
```
Linking to #50319, which is the general issue tracking this problem.